src/main/resources/swagger/api-docs.yaml

```diff
@@ -4115,10 +4115,30 @@ definitions:
       ok:
         type: boolean
         description: whether any system(s) need attention
-
+        default: false
       systems:
         type: object
         description: Map[String, SubsystemStatus]
+        default: {
```

Comment:
We do this in a couple places and it does work, but those aren't great reasons.

It does seem helpful to me that the response example in the codebase looks exactly like the live version, but I'm happy to switch it over if people feel strongly.